packages/peregrine/lib/talons/CartPage/ProductListing/useProduct.js

```diff
@@ -5,7 +5,7 @@ import { useCartContext } from '@magento/peregrine/lib/context/cart';
 import { useUserContext } from '@magento/peregrine/lib/context/user';
 import configuredVariant from '@magento/peregrine/lib/util/configuredVariant';
 
-import { useWishlist } from '../../Wishlist/Wishlist/useWishlist';
+import { useWishlist } from '../../Wishlist/useWishlist';
```

Comment:
I am still not sure where I am should put these hooks. So I just flattened them into the Wishlist talon in peregrine. Any help?

Reply:
I think this is fine for now - we have a followup PWA-1709 to consolidate.